### ISSUE TITLE
Create Cluster View

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#38]
## User story
1. As a user
2. I need a cluster view
3. So that I can see relevant information about all clusters

## Acceptance criteria
* [x] Create the cluster view component that visualizes a list of clusters according to the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) (for now these information should only be dummy data)
* [ ] Create a filter dropdown which provides a search box and checkboxes for all availables clusters 
=> only selected cluster shall be displayed (default is all)
* [x]  When a user clicks on a cluster this cluster gets selected as filter, the user is redirected to the namespace view, the cluster filter is passed on

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
